src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessrisksandneedshandoverservice/validators/AppSecAllowedCharacters.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.hmppsassessrisksandneedshandoverservice.validators
+
+import jakarta.validation.Constraint
+import jakarta.validation.Payload
+import jakarta.validation.ReportAsSingleViolation
+import jakarta.validation.constraints.Pattern
+import kotlin.reflect.KClass
+
+@Pattern(regexp = "^[a-zA-Z0-9-'\\s,./]+$")
```

Comment:
Is this right? On the ticket it says backslash, but then gives a forward slash as the example 😕